python/compiler/aiecc/main.py

```diff
@@ -694,6 +694,7 @@ async def run_flow(self):
         if(opts.unified):
           self.file_opt_with_addresses = os.path.join(self.tmpdirname, 'input_opt_with_addresses.mlir')
           await self.do_call(progress_bar.task, ['aie-opt', '--aie-localize-locks',
+                              '--aie-normalize-address-spaces',
```

Comment:
This is already part of aie-opt-passes?

Reply:
Yes, but it was suggested that this has to run before conversion to standard based on an error with handle address space 2

Reply:
Thanks, After your patch, I went back and actually processed what Jeff had indicated.

Reply:
It doesn't make sense to me to run this pass twice... so I factored it out of aie_opt_passes